### DISCUSSION TITLE
Nuovo impianto di casa. Consigli su componenti di gestione domotica.

### DISCUSSION THREAD

**Discussion by Pajaura** [indomus/forum#2025] [GENERALE - Discussioni generiche]
Ciao a tutti. Chiedo un supporto con consigli spassionati. Dopo taaaanti mesi di imprecazioni santificate o meno, sono quasi giunto (tra circa un mesetto) alla fase di realizzazione dell'impianto elettrico della mia abitazione (che non ne è assolutamente provvisto perché si tratta di un restauro da rustico di un antico fienile, quindi priva in origine di scopo abitativo e di qualsiasi impianto idraulico ed elettrico).
Avrei diverse questioni da porre, se qualche anima pia può concedermi tempo, esperienza e conoscenza, partendo dal presupposto che trattandosi di un impianto ex-novo vorrei evitare di incorrere in accrocchi improvvisati in stile "vorrei rendere il mio impianto tradizionale domotico".

1. Vorrei avere la gestione tradizionale e domotica delle illuminazioni dei locali. Non vorrei per utilizzare prodotti commerciali quali BT Living Now, Vimar ByMe e affini. Stavo pensando quindi di adoperare degli Shelly 4 PRO PM direttamente nei due quadri (uno per piano per meglio gestire le dorsali ed abbattere il surriscaldamento dei componenti altrimenti troppo assembrati). È una scelta condivisibile o mi conviene puntare su una soluzione diversa? Gli Shelly 4 PRO PM li ho scelti per via del collegamento ethernet che ridurrebbe a zero la latenza (ed è chiaramente più stabile rispetto a wifi, zwave, mqtt, ecc. nonchè privo di interferenze elettromagnetiche), perché facilmente integrabili e gestibili da HA e perché mi consentono il monitoraggio degli assorbimenti e la possibilità di attivare e disattivare gli switch anche direttamente nel quadro.
2. L’impianto di raffrescamento e riscaldamento, per il quale i lavori partiranno a inizio settimana prossima, sarà a pavimento. È previsto quindi che ci siano due deumidificatori (uno per piano) e che ogni stanza sia dotata di termostato per il controllo di temperature e umidità per la gestione a zone. Ovviamente i costi dei sensori (termostati) che mi sono stati inseriti nel computo sono mostruosi, parliamo di 250€ cad. Ora, esistono soluzioni più economiche ed integrabili nell’ecosistema di HA? Vorrei evitare comunque sensorini da 30€ tipo sonoff, aquara, ecc. perché sono virtualmente inaffidabili, tre identici uno a fianco all’altro danno valori sballati anche di due gradi e l'abitazione non è un giocattolo per il quale ci si può permettere di avere un così grossolano scarto di informazioni ambientali con relativa errata gestione dell'impianto.
3. I “termostati” posti nei locali dovranno poi comunicare con un termostato principale o sono di norma tutti indipendenti ed ognuno gestisce una termovalvola? Il termotecnico non è particolarmente ferrato in domotica (così come non lo sono io sotto questo aspetto) e quindi credo abbia optato per la soluzione più facile per tutti ma più onerosa per me. Nel primo caso, a vostro parere o per vostra esperienza, ce n’è uno maggiormente indicato e consigliato a tale scopo e per un miglior interfacciamento con HA? Nel secondo caso mi riallaccerò al punto 2.
4. Avrò diverse finestre che integrano veneziane o plisse oscuranti e due frangisole impacchettabili esterni. Ovviamente la gestione manuale dei motori sarà gestita tramite il classico pulsante dimmer. Nel caso trovassi appoggio da parte vostra sull'utilizzo degli Shelly 4 PRO PM mi troverei con un problema, ovvero che questi non gestiscono prodotti dimmerabili (così viene specificato nel manuale tecnico). Volendo evitare per l'appunto accrocchi strani (da Shelly vengono indicati lo Shelly 2.5 o lo Shelly Plus 2PM per la gestione di questi dispositivi) come mi consigliate di procedere in tal senso? L'idea di mantenere anche questa gestione all'interno dei quadri elettrici la fa da padrone ma non saprei su cosa ripiegare.
5. Gli scuri saranno motorizzati con un sistema YSLO FLEX io di Somfy. I motori non saranno gestiti attraverso dimmer a parete ma da un pulsante a comando radio di Somfy (non esiste altro modo di gestire i motori da pulsante fisico). C'è modo di gestire il tutto da HA? Qualcuno ha avuto la mia stessa necessità?

Scusate se sono stato prolisso e grazie in anticipo!

**Comment by NdR91** [comment]
Ciao, cerco di rispondere per punti.

**1- Utilizzo di Shelly 4PM dislocati in 2 quadri.**
E' una scelta che condivido, ma vorrei fare delle precisazioni.
I sistemi "OEM" tipo Vimar e BTicino sono validissimi, funzionano con e senza "domotica" e, cosa da non trascurare, non avranno mai alcun problema di certificazione dell'impianto. Utilizzano ZigBee e pertanto sono compatibili con Home Assistant.
Personalmente, non apprezzo la loro "rigidità" (Shelly mette a disposizione molte più opzioni che, spesso, tornano alquanto utili) ed il costo non indifferente quando si va a creare da 0 un intero appartamento.
Io adotterò una soluzione molto simile alla tua: creerò dei "piccoli quadri" in ogni stanza dove alloggiare gli Shelly 4PM. La ragione è semplice: stendere ogni singola luce fino ad un quadro richiede tempi e costi che non ho ritenuto "ragionevoli". Una derivazione in più in ogni stanza ha costi del tutto trascurabili e utilizzando quelle con "barra din" integrata verrà comunque un lavoro piuttosto ordinato.
Il discorso LAN è relativo: la latenza di uno Shelly in WiFi è talmente ridotta che non ha senso nemmeno paragonarla all'utilizzo in LAN. Valuta bene la stesura del cavo LAN per ogni Shelly, perchè diventerà decisamente ingombrante e costoso. E' una scelta, non sbagli in entrambi i casi a mio parere.

**2- Sensoristica / Termostati.**
Questo è un discorso complesso, perchè devi prima di tutto decidere se l'integrazione del riscaldamento/raffrescamento ad HA deve rappresentare l'unico accesso che hai al tuo impianto o un "di più".
Nel primo caso, puoi sicuramente sbizzarrirti e decidere in totale autonomia gli accessori. Mi sento di dissentire sulla qualità degli Aqara, che io invece ho visto essere piuttosto costante ed affidabile.
Nel secondo caso, la scelta di tale componentistica non è "libera" ed anzi, dovrai capire con i tuoi tecnici quali soluzioni "compatibili" tu possa usare. In generale, e lo dico basandomi sulle mia ricerche attuali (sono nella tua stessa identica situazione, ma i lavori inizieranno a Febbraio), le mandate delle zone non sono altro che un "on/off". Per cui virtualmente qualsiasi termostato (smart e non), se non addirittura un relè come uno Shelly, possono andare bene come soluzione. L'unico "vincolo" è mantenere in funzione il Cronotermostato OEM, che resta l'unico punto di accesso alle impostazioni della tua Pompa di Calore. Nulla vieta di metterlo, ad esempio, affianco al quadro di casa come appunto pannello di controllo della PDC e poi mettere nel resto delle stanze qualcosa a tua scelta.

Questa è la soluzione che dovrei adottare io. Personalmente ho scelto Tado° come linea di termostati.
Tutto ciò è in fase di verifica, sto aspettando che l'idraulico mi confermi che tutto questo funzioni come previsto.

**3- Termostati**
I Termostati posti nelle varie zone DOVREBBERO, come detto al punto 2, essere degli "on/ff" che regolano la loro mandata (o zona). In teoria non c'è comunicazione tra loro ed il Cronotermostato principale.

**4- Tapparelle**
Shelly 2.5 è la scelta più giusta secondo me. Ho già eseguito 3 installazioni in 3 abitazioni differenti (per amici) e funzionano davvero bene. Starà a te decidere se stendere anche questi fili fino al quadro o se gestirli con una derivazione li vicino.
Anche in questo caso, io ho optato per la derivazione. Pretendo un impianto ordinato, ma sinceramente non mi va nemmeno di spendere il 30% in più per stendere km di cavo aggiuntivi.

**5- Somfy** 
Ho visto che esiste un'integrazione Somfy, ma non so altro. Io l'ho proprio "evitata", scegliendo un sistema differente per gestire il cancello automatico.

Spero di averti aiutato, so di non dare "certezze" sul fronte termostati, ma anche io sono in una fase di progettazione.

**Comment by antony84** [comment]
Ciao, anche io mi trovo in una situazione simile, impianto di casa nuova.
Il dubbio principale è se usare i dispositivi PRO a quadro, centralizzando, o se usate i PLUS nella cassette 50x o nelle scatole di derivazione decentralizzando.

Le questioni diciamo sono 2:
- economica: un pro 4 pm costa il 70% in più di due plus 2 pm (circa 60 euri contro 100), e su 40 gruppi luci come nel mio caso fa la differenza. Oltre al fatto (spesa extra) che vanno portate tutte le luci al QG, comprese le strisce led a questo punto, anche se mi sembra che non ci siano dispositivi PRO dimmer.
- 
- stabilità e "salubrità" connessione: wifi vs ethernet

Cosa mi consigliate di fare? Vale davvero la pena spendere di più per i PRO?
